packages/KPrompt/KPrompt.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <KModal
+    :is-visible="isVisible"
+    :title="displayTitle"
+    class="k-prompt"
+    @keyup.esc="close">
+    <template v-slot:header-content>
+      <div class="k-prompt-header w-100">
+        <div class="k-prompt-header-content d-flex w-100">
+          <slot name="header-content">
+            <KIcon
+              v-if="type === 'warning'"
+              icon="warning"
+              color="var(--white)"
+              secondary-color="var(--yellow-400)"
+              size="20"
+              class="mr-2" />
+            {{ displayTitle }}
+          </slot>
+          <div class="close-button">
+            <KButton
+              class="non-visual-button"
+              @click="close">
+              <KIcon
+                icon="close"
+                color="var(--grey-600)"
+                size="15" />
+            </KButton>
+          </div>
+        </div>
+        <hr class="divider">
+      </div>
+    </template>
+    <template v-slot:body-content>
+      <div class="k-prompt-body">
+        <div class="k-prompt-body-content">
+          <slot name="body-content">
+            {{ message }}
+          </slot>
+
+          <div
+            v-if="confirmationText"
+            class="k-prompt-confirm-text w-100">
+            Type "<span class="bold-600">{{ confirmationText }}</span>" to confirm your action.
+
+            <KInput
+              v-model.trim="confirmationInput"
```

Comment:
When on the docs page, if I type in the `I Agree` text into the Prompt, then click `OK` to proceed, the modal closes correctly; however, if I open the same prompt again, the `I Agree` text is already inside of the `KInput`

![image](https://user-images.githubusercontent.com/2229946/154745940-5d2d969c-03be-45c5-b168-bd24568b472a.png)

We should clear the value of `confirmationInput` before the open action actually shows the Prompt.

Reply:
Fixed.